README.md

```diff
@@ -1,3 +1,12 @@
+![Scaleway Logo](./static/images/scaleway-logo.svg)
+
+
+![Codecov](https://img.shields.io/codecov/c/github/scaleway/scaleway-lib)
+![GitHub closed issues](https://img.shields.io/github/issues-closed/scaleway/scaleway-lib)
+![dependencies](https://david-dm.org/scaleway/scaleway-lib.svg)
```

Comment:
This one doesn't work :)

Reply:
It's work fine for me 
<img width="1064" alt="Capture d’écran 2021-03-16 à 10 35 10" src="https://user-images.githubusercontent.com/14060273/111287347-6deb4c80-8643-11eb-9d07-40b22cf08ee0.png">